test/correctness/simd_op_check.cpp

```diff
@@ -254,7 +226,43 @@ struct Test {
 
         bool can_run_the_code = can_run_code();
         if (can_run_the_code) {
-            Realization r = error.realize(target.without_feature(Target::NoRuntime));
+            Target run_target = target;
+            run_target.set_feature(Target::NoRuntime, false);
```

Comment:
style nit:  might be easier to read as a chained series of `without_feature()` calls?